src/actions/index.js

```diff
@@ -0,0 +1,5 @@
+import * as userLocation from './userLocation';
+
+export default {
+  ...userLocation
+};
```

Comment:
NOTE: for now we use the spread operator to flatten the tree an export all userLocation actions directly to the top of the tree (since they are called directly on Root and nowhere else). If we were to change our mind down the line, we could not use the spread operator here and instead write:

```javascript
export  default {
  userLocation,
  anonymousLocations
}
```

to have access to `this.props.userLocation` (which could contain several actions as k/v pairs -- currently only `userLocationChanged `) and `this.props.anonymousLocations` (which would also contain an object referencing several actions), and pass those around to separate child modules as desired....

The disruption of the state and action trees caused by such changes would be caught by, and require modifications to the tests in `reducers/index.spec.js` and `actions/index.spec.js`